docs/diagrams/LogicClassDiagram.puml

```diff
@@ -50,7 +50,6 @@ ArgumentTokenizer .left.> ArgumentMultimap
 XYZCommandParser ..> CliSyntax
 CliSyntax ..> Prefix
 XYZCommandParser ..> ParserUtil
-ParserUtil .down.> Prefix
```

Comment:
I changed this because I noticed that ParserUtil does not directly depend on Prefix.